Rdev/xgx_ordinal_regplot.R

```diff
@@ -0,0 +1,78 @@
+#' @importFrom brms brm
+#' @importFrom brms 
+#' @export
+xgx_ordinal_regression_plot <- function(data = NULL,
```

Comment:
Not sure difference between xgx_ordinal_regplot.R and xgx_ordinal_reg.R and xgx_ordinal_regression_plot.R

Reply:
I mean, in the end we should have only one, right?